### PR TITLE
feat(outputs.mqtt): Add sprig functions for topic name generator

### PR DESCRIPTION
**Pull request by diorcety**
## Summary
Simple minor improvement allow to use sprig functions inside topic name templates

### CODE REVIEW

File: plugins/outputs/mqtt/topic_name_generator.go [hunk 1]
```diff
@@ -5,6 +5,7 @@ import (
 	"strings"
 	"text/template"
 
+	"github.com/Masterminds/sprig/v3"
```
Comment (srebhan) [inline plugins/outputs/mqtt/topic_name_generator.go:8]:
```suggestion
	"github.com/Masterminds/sprig/v3"
	
```

**Review by srebhan — COMMENTED** [review]
Thanks @diorcety for your contribution. Just one style comment from my side...

**Review by srebhan — APPROVED** [review]

**Comment by srebhan** [comment]
@diorcety could you please run `make fmt` to make CI happy?!

**Comment by diorcety** [comment]
@srebhan Done

**Review by DStrand1 — APPROVED** [review]

**Pull request merged**